[13.0][MIG] crm_claim_type: Migration to 13.0 (#339)
OCA/crm: crm_claim_type/readme/DESCRIPTION.rst

> Includes a way to classify claims adding the concept of type and with this,
> it allows to configure claim stages depending on claim types.
> 
> This module includes Customer, Supplier and Other claim types as default data
> and its own stages relation, and also makes few stages common for different
> claim types

Reviewer: ```suggestion
Provides a way to classify claims by type and to configure claim stages depending on claim types.

Includes Customer, Supplier and Other claim types by default,
as well as stages configuration.
```
